src/components/modals/edit-movie/edit-movie.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
+import { useLoaderData } from 'react-router-dom';
 import Dialog from 'src/components/dialog/dialog';
 import MovieForm from 'src/components/movie-form/movie-form';
+import { DIALOG } from 'src/constants/dialog-type.constants';
+
+const EditMovie = () => {
+  const movie = useLoaderData();
 
-const EditMovie = ({ movie }) => {
   return(
-    <Dialog title={'Edit Movie'} onClose={() => {}}>
-      <MovieForm movie={movie}/>
+    <Dialog title={DIALOG.TITLE.EDIT_MOVIE}>
+      <MovieForm movie={movie} type={DIALOG.TYPE.EDIT}/>
```

Comment:
В задании было так написано, поэтому и не трогала параметры: "Optionally, you can preserve current search query parameters, so that movie list is not updated when you click on the link."